### PR TITLE
[PR-1] added Jupyter Notebook dependency for installation

### PR DESCRIPTION
**Pull request by jonatng**
Jupyter Notebook installation is needed when installing promptTools

### CODE REVIEW

**Comment by HashemAlsaket** [comment]
Awesome catch

**Comment by jonatng** [comment]
Hi @NivekT @steventkrawczyk 
Can I get some eyes on this PR please

**Review by NivekT — CHANGES_REQUESTED** [review]
Thanks for opening this PR! 

I think it should be added to the Quick Start section, possibly here:

```
cd prompttools && jupyter notebook examples/notebooks/OpenAIChatExperiment.ipynb
```

**Review by NivekT — CHANGES_REQUESTED** [review]
The change should still be in README but below
```bash
You can run a simple example of a `prompttools` locally with the following

```

**Review by NivekT — CHANGES_REQUESTED** [review]
`pip install notebook` should be in README not in a single Jupyter notebook

**Comment by jonatng** [comment]
@NivekT  Added pip install notebook to README
Would you like to keep or remove pip install notebook from _prompttools/examples/notebooks/OpenAIChatExperiment.ipynb_

File: README.md [hunk 1]
```diff
@@ -69,7 +69,11 @@ You can also run the notebook in [Google Colab](https://colab.research.google.co
 
 If you want to interact with `prompttools` using our playground interface, you can launch it with the following commands.
 
-First, install prompttools:
+You can run a simple example of a `prompttools` locally with the following
+
+```
+pip install notebook
+```
```
Comment (NivekT) [inline README.md:76]:
```suggestion
pip install notebook  # If jupyter notebook has not been installed
```

File: examples/notebooks/OpenAIChatExperiment.ipynb [hunk 2]
```diff
@@ -23,6 +23,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "# pip install notebook\n",
```
Comment (NivekT) [inline examples/notebooks/OpenAIChatExperiment.ipynb:26]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -69,7 +69,10 @@ You can also run the notebook in [Google Colab](https://colab.research.google.co
 
 If you want to interact with `prompttools` using our playground interface, you can launch it with the following commands.
 
-First, install prompttools:
+You can run a simple example of a `prompttools` locally with the following
+
+```
+pip install notebook  # If jupyter notebook has not been installed
```
Comment (NivekT) [inline README.md:75]:
```suggestion
pip install notebook  # If jupyter notebook has not been installed
```
```

**Pull request merged**